backend/libs/room-operations.ts

```diff
@@ -83,6 +83,37 @@ export const createRoom = async (
     }
 };
 
+export const updateRoom = async (
+    room: Room,
+    tableName: string,
+    dynamoDb: DocumentClient = new DocumentClient(),
+): Promise<Room | undefined> => {
+    const roomForDDB = marshallRoom(room);
```

Comment:
Do we need to marshal it if we only update a few fields ?

Reply:
Updating only relevant fields would require more code complexity or relying on an ORM that can do that.